dash/dash.py

```diff
@@ -631,6 +631,7 @@ def dependencies(self):
                 'output': k,
                 'inputs': v['inputs'],
                 'state': v['state'],
+                'clientside_function': v.get('clientside_function', {})
```

Comment:
Personally I'd default to `None` rather than `{}`. It seems clearer and the logic is simpler (no need for the `isEmpty` clause)